store/index.js

```diff
@@ -75,10 +70,6 @@ const createStore = () => new Vuex.Store({
         commit('SITE_LOADED')
       }, 1500)
     }
```

Comment:
You need the namescaced attribute in order to use modules. For instance your scroll trigger you need to use it in the index store.

Reply:
I removed this line because by default nuxt compiles the modules as namespaced :)

Reply:
@troitroi Ok I'll let you pull and fix the conflicts if there are any